validator/graffiti/parse_graffiti_test.go

```diff
@@ -123,7 +124,7 @@ ordered:
   - "Mr F was here"
 
 specific:
-  1234: Yolo
+  1234: "hex:796f6c6f"
   555: "What's up"
   703727: Meow`)
```

Comment:
Please don't change an existing test. This previous logic should still behave the same. Add a new test for `hex:` prefixed strings.